HULKs/hulk
# Hula next

## Introduced Changes

Removes the aliveness capabilities from hula.
Aliveness will be added in a future PR. (probably #27)

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

- implement aliveness

## How to Test

- flash an image with this hula (ask @schmidma)
- upload the code, see if everything still works
